src/main/java/movie/storage/exception/IncorrectDataHandlingException.java

```diff
@@ -0,0 +1,7 @@
+package movie.storage.exception;
+
+public class IncorrectDataHandlingException extends RuntimeException {
```

Comment:
exception's cause its incorrect data or incorrect way of handling data?

Reply:
Changed to IncorrectDataException